packages/dd-trace/src/config.js

```diff
@@ -788,6 +790,7 @@ class Config {
       this._setValue(env, 'iast.requestSampling', iastRequestSampling)
     }
     this._envUnprocessed['iast.requestSampling'] = DD_IAST_REQUEST_SAMPLING
+    this._setString(env, 'iast.securityControlsConfiguration', DD_IAST_SECURITY_CONTROLS_CONFIGURATION)
```

Comment:
you are not giving an option to configure it programatically, intentionally or you just forgot it?

Reply:
added